package.json

```diff
@@ -94,12 +94,12 @@
     "@types/update-notifier": "^1.0.2",
     "@types/ws": "^4.0.1",
     "@types/yargs": "^8.0.2",
+    "@wardpeet/brfs": "^2.1.0-0",
```

Comment:
@wardpeet did you rename/republish to wardpack?

Reply:
no need, but if you want to ;)

I was mostly making sure you didn't publish to a new package + possible updates